decred/decred/crypto/secp256k1/curve.py

```diff
@@ -337,41 +378,47 @@ def publicKey(self, k):
         x, y = self.scalarBaseMult(k)
         return PublicKey(self, x, y)
 
-    def parsePubKey(self, pubKeyStr):
+    def parsePubKey(self, pubKey):
         """
```

Comment:
We have a class in this module called `PublicKey`, so `pubKey` is not great. I've regularly named byte-array-like things with a trailing capital B, so `pubKeyB`.